templates/cpa-cpu.yaml.tpl

```diff
@@ -0,0 +1,74 @@
+affinity: {}
+config: 
+#  ladder:
+#    coresToReplicas:
+#      - [ 1, 1 ]
+#      - [ 64, 3 ]
+#      - [ 512, 5 ]
+#      - [ 1024, 7 ]
+#      - [ 2048, 10 ]
+#      - [ 4096, 15 ]
+#    nodesToReplicas:
+#      - [ 1, 1 ]
+#      - [ 2, 2 ]
+  linear:
+    nodesPerReplica: 1
+    min: 1
+    max: 100
```

Comment:
Is this max too high?

Reply:
Its a ceiling value, that we may not (or may we?) ever reach.